static/js/reducers/videoUi.js

```diff
@@ -20,10 +22,11 @@ export const INITIAL_UPLOAD_SUBTITLE_FORM_STATE = {
 };
 
 export const INITIAL_UI_STATE = {
-  videoSubtitleForm: INITIAL_UPLOAD_SUBTITLE_FORM_STATE
+  videoSubtitleForm: INITIAL_UPLOAD_SUBTITLE_FORM_STATE,
+  corner: 'upperLeft'
```

Comment:
Can you make these values (`upperLeft` and etc) into constants and put them in `constants.js`?

Reply:
OK, I put this in `constants.py`:

```python
export const CANVASES = {
  upperLeft: {shiftX: false, shiftY: false},
  upperRight: {shiftX: true, shiftY: false},
  lowerLeft: {shiftX: false, shiftY: true},
  lowerRight: {shiftX: true, shiftY: true}
};
```

So `videoUI.js` above would be changed to `corner: CANVASES[upperLeft]`: